interface/super/edit_globals.php

```diff
@@ -645,6 +645,22 @@ function checkBackgroundServices()
                                                     echo "</option>\n";
                                                 }
                                                 echo "  </select>\n";
+                                            } elseif ($fldtype == GlobalSetting::DATA_TYPE_MULTI_DASHBOARD_CARDS) {
+                                                $res = sqlStatement("SELECT * FROM dashboard_card_visibility");
```

Comment:
You were supposed to get rid of table and hard code these options!

Reply:
@sjpadgett eye eye captain